easybuild/easyblocks/generic/configuremake.py

```diff
@@ -79,6 +79,9 @@ def configure_step(self, cmd_prefix=''):
             for (key, val) in tar_vars.items():
                 self.cfg.update('preconfigopts', "%s='%s'" % (key, val))
 
+         # Ensure that configure has the excutable bit set, needed if a patch created configure itself
+         import os 
+         os.chmod('./configure', 0755)
```

Comment:
please use the `adjust_permissions` function available from `easybuild.tools.filetools` for this, something like:

```
adjust_permissions('configure', stat.S_IXUSR, add=True)
```

and only import additional stuff at the top of the easyblock, where the other imports are; you'll need to add:

```python
import stat
...
from easybuild.tools.filetools import adjust_permissions
```